main/src/ca/uwaterloo/flix/language/phase/Eraser.scala

```diff
@@ -3,11 +3,13 @@ package ca.uwaterloo.flix.language.phase
 import ca.uwaterloo.flix.api.Flix
 import ca.uwaterloo.flix.language.ast.SimpleType.erase
 import ca.uwaterloo.flix.language.ast.{AtomicOp, ErasedAst, Purity, ReducedAst, SimpleType, SourceLocation, Symbol, Type, TypeConstructor}
-import ca.uwaterloo.flix.language.dbg.AstPrinter.{DebugNoOp, DebugReducedAst}
-import ca.uwaterloo.flix.util.collection.MapOps
+import ca.uwaterloo.flix.language.dbg.AstPrinter.DebugNoOp
+import ca.uwaterloo.flix.util.collection.ListOps
 import ca.uwaterloo.flix.util.{InternalCompilerException, ParOps}
 
+import java.util.concurrent.ConcurrentHashMap
 import scala.annotation.unused
+import scala.jdk.CollectionConverters.ConcurrentMapHasAsScala
```

Comment:
Where is this used? We don't want Scala concurrent maps. It should be a proper Scala map, not a Scala concurrent map.

Reply:
I note it is also used in Inliner, but that is also wrong.

Reply:
We don't use the concurrent version anywhere else.